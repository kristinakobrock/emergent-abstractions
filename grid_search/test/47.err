  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  4%|###7                                                                                         | 2/50 [00:00<00:02, 16.75it/s]  8%|#######4                                                                                     | 4/50 [00:00<00:02, 16.80it/s] 14%|#############                                                                                | 7/50 [00:00<00:02, 19.87it/s] 18%|################7                                                                            | 9/50 [00:00<00:02, 17.32it/s] 24%|######################                                                                      | 12/50 [00:00<00:01, 19.19it/s] 30%|###########################6                                                                | 15/50 [00:00<00:01, 19.79it/s] 36%|#################################1                                                          | 18/50 [00:00<00:01, 21.95it/s] 42%|######################################6                                                     | 21/50 [00:01<00:01, 21.99it/s] 48%|############################################1                                               | 24/50 [00:01<00:01, 21.16it/s] 54%|#################################################6                                          | 27/50 [00:01<00:01, 21.41it/s] 62%|#########################################################                                   | 31/50 [00:01<00:00, 24.88it/s] 68%|##############################################################5                             | 34/50 [00:01<00:00, 25.04it/s] 74%|####################################################################                        | 37/50 [00:01<00:00, 26.23it/s] 82%|###########################################################################4                | 41/50 [00:01<00:00, 28.60it/s] 88%|################################################################################9           | 44/50 [00:01<00:00, 27.65it/s] 94%|######################################################################################4     | 47/50 [00:01<00:00, 28.27it/s]100%|############################################################################################| 50/50 [00:02<00:00, 24.06it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 23%|#####################4                                                                       | 3/13 [00:00<00:00, 25.42it/s] 54%|##################################################                                           | 7/13 [00:00<00:00, 28.03it/s] 85%|#############################################################################8              | 11/13 [00:00<00:00, 28.58it/s]100%|############################################################################################| 13/13 [00:00<00:00, 26.81it/s]
