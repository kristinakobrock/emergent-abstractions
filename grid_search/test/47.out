# ["--attributes=3", "--values=16", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=128", "--learning_rate=0.001", "--hidden_size=256", "--temp_update=0.99", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.8208694458007812, "acc": 0.5766878724098206, "length": 3.4198174476623535, "mode": "train", "epoch": 1}
{"loss": 0.6651580333709717, "acc": 0.623242199420929, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6969424486160278, "acc": 0.5171529650688171, "length": 1.907380223274231, "mode": "train", "epoch": 2}
{"loss": 0.6646472215652466, "acc": 0.6230469346046448, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6724290251731873, "acc": 0.6265746355056763, "length": 1.0874474048614502, "mode": "train", "epoch": 3}
{"loss": 0.6687934398651123, "acc": 0.6230469346046448, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6637107133865356, "acc": 0.6241757869720459, "length": 1.0333178043365479, "mode": "train", "epoch": 4}
{"loss": 0.6683444976806641, "acc": 0.6146484017372131, "length": 1.0, "mode": "test", "epoch": 4}
{"loss": 0.6616643071174622, "acc": 0.6249824166297913, "length": 1.021241545677185, "mode": "train", "epoch": 5}
{"loss": 0.6658681631088257, "acc": 0.6230469346046448, "length": 1.0, "mode": "test", "epoch": 5}
{"loss": 0.6601570248603821, "acc": 0.6269799470901489, "length": 1.016467809677124, "mode": "train", "epoch": 6}
{"loss": 0.6638200283050537, "acc": 0.6210936903953552, "length": 1.0, "mode": "test", "epoch": 6}
{"loss": 0.6593693494796753, "acc": 0.6261758804321289, "length": 1.0123594999313354, "mode": "train", "epoch": 7}
{"loss": 0.6628923416137695, "acc": 0.6230469346046448, "length": 1.0, "mode": "test", "epoch": 7}
{"loss": 0.6588508486747742, "acc": 0.6264106035232544, "length": 1.010574460029602, "mode": "train", "epoch": 8}
{"loss": 0.6619236469268799, "acc": 0.6248047351837158, "length": 1.0, "mode": "test", "epoch": 8}
{"loss": 0.6583857536315918, "acc": 0.6266430616378784, "length": 1.008513331413269, "mode": "train", "epoch": 9}
{"loss": 0.6620432138442993, "acc": 0.6234374642372131, "length": 1.0, "mode": "test", "epoch": 9}
{"loss": 0.6583076119422913, "acc": 0.6266982555389404, "length": 1.0075006484985352, "mode": "train", "epoch": 10}
{"loss": 0.6615877151489258, "acc": 0.6224609017372131, "length": 1.0, "mode": "test", "epoch": 10}
{"loss": 0.6580319404602051, "acc": 0.6266974806785583, "length": 1.006804347038269, "mode": "train", "epoch": 11}
{"loss": 0.6616209149360657, "acc": 0.6234374642372131, "length": 1.0, "mode": "test", "epoch": 11}
{"loss": 0.6577103734016418, "acc": 0.6269298195838928, "length": 1.0055934190750122, "mode": "train", "epoch": 12}
{"loss": 0.6613966226577759, "acc": 0.623828113079071, "length": 1.0, "mode": "test", "epoch": 12}
{"loss": 0.6575697064399719, "acc": 0.6264720559120178, "length": 1.0047558546066284, "mode": "train", "epoch": 13}
{"loss": 0.6616380214691162, "acc": 0.6236328482627869, "length": 1.0, "mode": "test", "epoch": 13}
{"loss": 0.6575393676757812, "acc": 0.6270410418510437, "length": 1.0041379928588867, "mode": "train", "epoch": 14}
{"loss": 0.6616790294647217, "acc": 0.6214843392372131, "length": 1.0, "mode": "test", "epoch": 14}
{"loss": 0.657089352607727, "acc": 0.6272739171981812, "length": 1.003631591796875, "mode": "train", "epoch": 15}
{"loss": 0.6620926260948181, "acc": 0.6234374642372131, "length": 1.0, "mode": "test", "epoch": 15}
{"loss": 0.6576007008552551, "acc": 0.6265844702720642, "length": 1.0033758878707886, "mode": "train", "epoch": 16}
{"loss": 0.6612932682037354, "acc": 0.6232421398162842, "length": 1.0, "mode": "test", "epoch": 16}
{"loss": 0.6566879153251648, "acc": 0.6269872784614563, "length": 1.0030617713928223, "mode": "train", "epoch": 17}
{"loss": 0.6615157127380371, "acc": 0.6236328482627869, "length": 1.0, "mode": "test", "epoch": 17}
{"loss": 0.6568343043327332, "acc": 0.6271572113037109, "length": 1.0028417110443115, "mode": "train", "epoch": 18}
{"loss": 0.6614421606063843, "acc": 0.6220703125, "length": 1.0, "mode": "test", "epoch": 18}
{"loss": 0.6569846868515015, "acc": 0.6271023154258728, "length": 1.0022544860839844, "mode": "train", "epoch": 19}
{"loss": 0.6618123054504395, "acc": 0.6238281726837158, "length": 1.0, "mode": "test", "epoch": 19}
{"loss": 0.6565425992012024, "acc": 0.6267547607421875, "length": 1.002170205116272, "mode": "train", "epoch": 20}
{"loss": 0.6617106795310974, "acc": 0.6236328482627869, "length": 1.0, "mode": "test", "epoch": 20}
{"loss": 0.6565049290657043, "acc": 0.6269270181655884, "length": 1.0018759965896606, "mode": "train", "epoch": 21}
{"loss": 0.6619695425033569, "acc": 0.6214843392372131, "length": 1.0, "mode": "test", "epoch": 21}
{"loss": 0.6572350263595581, "acc": 0.6276171207427979, "length": 1.0016008615493774, "mode": "train", "epoch": 22}
{"loss": 0.66195148229599, "acc": 0.6236328482627869, "length": 1.0, "mode": "test", "epoch": 22}
{"loss": 0.6563103795051575, "acc": 0.6269843578338623, "length": 1.0014979839324951, "mode": "train", "epoch": 23}
{"loss": 0.6618662476539612, "acc": 0.6234374642372131, "length": 1.0, "mode": "test", "epoch": 23}
{"loss": 0.6566517949104309, "acc": 0.6265833973884583, "length": 1.0014808177947998, "mode": "train", "epoch": 24}
{"loss": 0.6622757911682129, "acc": 0.6224609017372131, "length": 1.0, "mode": "test", "epoch": 24}
{"loss": 0.6556001305580139, "acc": 0.6280772089958191, "length": 1.0013220310211182, "mode": "train", "epoch": 25}
{"loss": 0.6627647280693054, "acc": 0.6234374642372131, "length": 1.0, "mode": "test", "epoch": 25}
{"loss": 0.6564554572105408, "acc": 0.6272727847099304, "length": 1.0012723207473755, "mode": "train", "epoch": 26}
{"loss": 0.6624665260314941, "acc": 0.6226562857627869, "length": 1.0, "mode": "test", "epoch": 26}
{"loss": 0.6566339135169983, "acc": 0.6270430088043213, "length": 1.0010894536972046, "mode": "train", "epoch": 27}
{"loss": 0.662239134311676, "acc": 0.6248047351837158, "length": 1.0, "mode": "test", "epoch": 27}
{"loss": 0.6561285853385925, "acc": 0.6273303031921387, "length": 1.0009652376174927, "mode": "train", "epoch": 28}
{"loss": 0.6632929444313049, "acc": 0.6236328482627869, "length": 1.0, "mode": "test", "epoch": 28}
{"loss": 0.6564555168151855, "acc": 0.6269272565841675, "length": 1.0008984804153442, "mode": "train", "epoch": 29}
{"loss": 0.6625969409942627, "acc": 0.6222655773162842, "length": 1.0, "mode": "test", "epoch": 29}
{"loss": 0.656779408454895, "acc": 0.627099871635437, "length": 1.0008100271224976, "mode": "train", "epoch": 30}
{"loss": 0.662859320640564, "acc": 0.6220703125, "length": 1.0, "mode": "test", "epoch": 30}
{"loss": 0.6565660834312439, "acc": 0.6276745796203613, "length": 1.000813603401184, "mode": "train", "epoch": 31}
{"loss": 0.663906455039978, "acc": 0.6236328482627869, "length": 1.0, "mode": "test", "epoch": 31}
{"loss": 0.6563494801521301, "acc": 0.6270996332168579, "length": 1.0007750988006592, "mode": "train", "epoch": 32}
{"loss": 0.66297847032547, "acc": 0.623828113079071, "length": 1.0, "mode": "test", "epoch": 32}
{"loss": 0.6559802293777466, "acc": 0.6269854307174683, "length": 1.0006687641143799, "mode": "train", "epoch": 33}
{"loss": 0.6625300049781799, "acc": 0.6232421398162842, "length": 1.0, "mode": "test", "epoch": 33}
{"loss": 0.6554588079452515, "acc": 0.6266975998878479, "length": 1.0005943775177002, "mode": "train", "epoch": 34}
{"loss": 0.6626766920089722, "acc": 0.6232421398162842, "length": 1.0, "mode": "test", "epoch": 34}
{"loss": 0.6559208631515503, "acc": 0.6271578073501587, "length": 1.0004757642745972, "mode": "train", "epoch": 35}
{"loss": 0.6627309322357178, "acc": 0.6224609017372131, "length": 1.0, "mode": "test", "epoch": 35}
{"loss": 0.656006395816803, "acc": 0.6270428895950317, "length": 1.0004510879516602, "mode": "train", "epoch": 36}
{"loss": 0.6623275279998779, "acc": 0.6240234375, "length": 1.0, "mode": "test", "epoch": 36}
{"loss": 0.6559177041053772, "acc": 0.6269848942756653, "length": 1.0004267692565918, "mode": "train", "epoch": 37}
{"loss": 0.6639269590377808, "acc": 0.6240234375, "length": 1.0, "mode": "test", "epoch": 37}
{"loss": 0.6560679078102112, "acc": 0.6266975402832031, "length": 1.0004072189331055, "mode": "train", "epoch": 38}
{"loss": 0.662676215171814, "acc": 0.6216797232627869, "length": 1.0, "mode": "test", "epoch": 38}
{"loss": 0.6553428769111633, "acc": 0.6271001696586609, "length": 1.0003907680511475, "mode": "train", "epoch": 39}
{"loss": 0.6628351211547852, "acc": 0.6224609017372131, "length": 1.0, "mode": "test", "epoch": 39}
{"loss": 0.6553031206130981, "acc": 0.6269275546073914, "length": 1.0003125667572021, "mode": "train", "epoch": 40}
{"loss": 0.6636266112327576, "acc": 0.6234374642372131, "length": 1.0, "mode": "test", "epoch": 40}
{"loss": 0.6558820605278015, "acc": 0.6275028586387634, "length": 1.0002943277359009, "mode": "train", "epoch": 41}
{"loss": 0.6625046133995056, "acc": 0.622851550579071, "length": 1.0, "mode": "test", "epoch": 41}
{"loss": 0.6548863053321838, "acc": 0.6272724866867065, "length": 1.000246524810791, "mode": "train", "epoch": 42}
{"loss": 0.6631049513816833, "acc": 0.6220702528953552, "length": 1.0, "mode": "test", "epoch": 42}
{"loss": 0.6553202271461487, "acc": 0.6273877620697021, "length": 1.0002328157424927, "mode": "train", "epoch": 43}
{"loss": 0.6627156138420105, "acc": 0.6224610209465027, "length": 1.0, "mode": "test", "epoch": 43}
{"loss": 0.6553527116775513, "acc": 0.6279051899909973, "length": 1.000264286994934, "mode": "train", "epoch": 44}
{"loss": 0.6632562875747681, "acc": 0.6238281726837158, "length": 1.0, "mode": "test", "epoch": 44}
{"loss": 0.6550682783126831, "acc": 0.6276752948760986, "length": 1.000234842300415, "mode": "train", "epoch": 45}
{"loss": 0.662673830986023, "acc": 0.6218750476837158, "length": 1.0, "mode": "test", "epoch": 45}
{"loss": 0.6550136804580688, "acc": 0.6270425319671631, "length": 1.0001654624938965, "mode": "train", "epoch": 46}
{"loss": 0.6630142331123352, "acc": 0.6218750476837158, "length": 1.0, "mode": "test", "epoch": 46}
{"loss": 0.6561523079872131, "acc": 0.6271000504493713, "length": 1.000185489654541, "mode": "train", "epoch": 47}
{"loss": 0.663356363773346, "acc": 0.6216797232627869, "length": 1.0, "mode": "test", "epoch": 47}
{"loss": 0.6555262804031372, "acc": 0.627848207950592, "length": 1.0001789331436157, "mode": "train", "epoch": 48}
{"loss": 0.662926197052002, "acc": 0.6236328482627869, "length": 1.0, "mode": "test", "epoch": 48}
{"loss": 0.6550390124320984, "acc": 0.6274449825286865, "length": 1.0001580715179443, "mode": "train", "epoch": 49}
{"loss": 0.6628493666648865, "acc": 0.6224609017372131, "length": 1.0, "mode": "test", "epoch": 49}
{"loss": 0.6549266576766968, "acc": 0.626812219619751, "length": 1.000113606452942, "mode": "train", "epoch": 50}
{"loss": 0.6633185148239136, "acc": 0.6222657561302185, "length": 1.0, "mode": "test", "epoch": 50}
{"loss": 0.6549294590950012, "acc": 0.6270425915718079, "length": 1.0001097917556763, "mode": "train", "epoch": 51}
{"loss": 0.6628737449645996, "acc": 0.6224609017372131, "length": 1.0, "mode": "test", "epoch": 51}
{"loss": 0.6545809507369995, "acc": 0.6276754140853882, "length": 1.0001250505447388, "mode": "train", "epoch": 52}
{"loss": 0.663232684135437, "acc": 0.6222655773162842, "length": 1.0, "mode": "test", "epoch": 52}
{"loss": 0.6553042531013489, "acc": 0.6277903914451599, "length": 1.0001060962677002, "mode": "train", "epoch": 53}
{"loss": 0.6630931496620178, "acc": 0.6216797232627869, "length": 1.0, "mode": "test", "epoch": 53}
{"loss": 0.6550818085670471, "acc": 0.6272151470184326, "length": 1.0000907182693481, "mode": "train", "epoch": 54}
{"loss": 0.6629329919815063, "acc": 0.6220703125, "length": 1.0, "mode": "test", "epoch": 54}
{"loss": 0.6554611921310425, "acc": 0.6273877620697021, "length": 1.000136375427246, "mode": "train", "epoch": 55}
{"loss": 0.6631320714950562, "acc": 0.621289074420929, "length": 1.0, "mode": "test", "epoch": 55}
{"loss": 0.654862105846405, "acc": 0.6269274353981018, "length": 1.0000824928283691, "mode": "train", "epoch": 56}
{"loss": 0.6636834144592285, "acc": 0.6222655773162842, "length": 1.0, "mode": "test", "epoch": 56}
{"loss": 0.6549624800682068, "acc": 0.6276754140853882, "length": 1.000059962272644, "mode": "train", "epoch": 57}
{"loss": 0.6633318662643433, "acc": 0.622265636920929, "length": 1.0, "mode": "test", "epoch": 57}
{"loss": 0.6549110412597656, "acc": 0.6265246868133545, "length": 1.000065565109253, "mode": "train", "epoch": 58}
{"loss": 0.6641696691513062, "acc": 0.622265636920929, "length": 1.0, "mode": "test", "epoch": 58}
{"loss": 0.6547864079475403, "acc": 0.6272150874137878, "length": 1.0000667572021484, "mode": "train", "epoch": 59}
{"loss": 0.6632333993911743, "acc": 0.6220703125, "length": 1.0, "mode": "test", "epoch": 59}
{"loss": 0.6548652648925781, "acc": 0.6277324557304382, "length": 1.0001541376113892, "mode": "train", "epoch": 60}
{"loss": 0.6637832522392273, "acc": 0.6232421398162842, "length": 1.0, "mode": "test", "epoch": 60}
test accuracy: 0.6330357193946838
